.github/workflows/e2e_tests.yml

```
@@ -4,139 +4,52 @@ on:
   deployment_status:
 
 jobs:
-  _e2e-tests:
+  liam-app-e2e_tests:
```

actionlint (1.7.7)

"steps" section is missing in job "liam-app-e2e_tests"

(syntax-check)